Mathlib/Geometry/Euclidean/Sphere/Basic.lean

```diff
@@ -46,9 +46,9 @@ open Module
 be positive; that should be given as a hypothesis to lemmas that require it. -/
 @[ext]
 structure Sphere [MetricSpace P] where
-  /-- center of this sphere -/
+  /-- Center of this sphere. -/
```

Comment:
Not sure about the period. @grunweg has told me to remove it in the past. Same below